packages/application/src/connectionlost.ts

```diff
@@ -4,7 +4,7 @@
 import { showErrorMessage } from '@jupyterlab/apputils';
 import { ServerConnection, ServiceManager } from '@jupyterlab/services';
 import { ITranslator, nullTranslator } from '@jupyterlab/translation';
-import { IConnectionLost } from './tokens';
+import { IConnectionLost } from './tokens.js';
```

Comment:
Looking at the diff, these `.js` suffixes looks a bit odd from `.ts` files.

Reply:
There is no other alternatives offered by TypeScript for now.